tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -631,6 +631,7 @@ public void AddRepositoryViaUri(string url, string cloneLocation, Frame loginFra
 
         EverythingToClone.Add(cloningInformation);
         ShouldEnablePrimaryButton = true;
+        ShouldShowUrlError = Visibility.Collapsed;

```

Comment:
Minor thing here: but I think we could actually change ShouldShowUrlError to a boolean and use it directly. Should still work without having to change anything on[ this line](https://github.com/microsoft/devhome/blob/9ec52bb99d48facd876817dd65a561eee87ab204/tools/SetupFlow/DevHome.SetupFlow/Views/AddRepoDialog.xaml#L60) I believe. This way we won't have to use the Visibility UI property directly in the view model.